tests/storage/migration.go

```diff
@@ -703,6 +717,41 @@ var _ = SIGDescribe("Volumes update with migration", decorators.RequiresTwoSched
 				libwait.WaitForVMIPhase(vmi, []v1.VirtualMachineInstancePhase{v1.Running})
 			})
 		})
+
+		It("should be able to migrate a VM with vTPM and the persistent state", func() {
+			volName := "volume"
+			dv := createDV()
+
+			vmi := libvmi.New(
+				libvmi.WithNamespace(ns),
+				libvmi.WithInterface(libvmi.InterfaceDeviceWithMasqueradeBinding()),
+				libvmi.WithNetwork(virtv1.DefaultPodNetwork()),
+				libvmi.WithResourceMemory("128Mi"),
+				libvmi.WithDataVolume(volName, dv.Name),
+				libvmi.WithCloudInitNoCloud(libvmifact.WithDummyCloudForFastBoot()),
+				libvmi.WithPersistentTPM(),
+			)
+			vm := libvmi.NewVirtualMachine(vmi,
+				libvmi.WithRunStrategy(virtv1.RunStrategyAlways),
+				libvmi.WithDataVolumeTemplate(dv),
+			)
+			vm, err := virtClient.VirtualMachine(ns).Create(context.Background(), vm, metav1.CreateOptions{})
+			Expect(err).ToNot(HaveOccurred())
+			Eventually(matcher.ThisVM(vm), 360*time.Second, 1*time.Second).Should(matcher.BeReady())
+			libwait.WaitForSuccessfulVMIStart(vmi)
+
+			destDV := createBlankDV(virtClient, ns, size)
+			By("Update volumes")
+			updateVMWithDV(vm, volName, destDV.Name)
```

Comment:
So IIUC this test is only migrating the cirros boot DV? not the backend PVC?

Reply:
Yes